main/streams/plain_wrapper.c

```diff
@@ -1341,7 +1341,7 @@ static int php_plain_files_metadata(php_stream_wrapper *wrapper, const char *url
 			break;
 		case PHP_STREAM_META_GROUP:
 		case PHP_STREAM_META_GROUP_NAME:
-			if(option == PHP_STREAM_META_OWNER_NAME) {
+			if(option == PHP_STREAM_META_GROUP_NAME) {
```

Comment:
At the stage, we're inside a "switch(option) {"
If we were PHP_STREAM_META_OWNER, we would have been executed at line#1332 -- which break;s' before it would reach line #1343